src/SwedbankPay/Api/Client/ClientVersion.php

```diff
@@ -172,27 +174,25 @@ private function tryGetVersionNumberFromComposerJson(&$version) : bool
       */
     private function tryGetVersionNumberFromComposerLock(&$version) : bool
     {
-        $composerLock = null;
+        $paths = [];
 
-        if (!$this->tryReadComposerLock($composerLock)) {
-            return false;
-        }
+        // Standard path of composer.lock
+        $path = $this->getComposerPath() . DIRECTORY_SEPARATOR . 'composer.lock';
+        $paths[] = $path;
 
-        if (isset($composerLock['packages'])) {
-            $packages = $composerLock['packages'];
-            foreach ($packages as $package) {
-                if (!isset($package['name']) ||
-                    $package['name'] !== 'swedbank-pay/swedbank-pay-sdk-php') {
-                    continue;
-                }
-
-                if (isset($package['version'])) {
-                    $version = $package['version'];
-
-                    if ($version !== null && !empty($version)) {
-                        return true;
-                    }
-                }
+        // Alternate path of composer.lock
+        $pathDirs = explode(DIRECTORY_SEPARATOR, $path);
+        array_splice($pathDirs, -2);
+        $pathDirs[] = 'composer.lock';
+        $paths[] = implode(DIRECTORY_SEPARATOR, $pathDirs);
+
+        foreach ($paths as $path) {
+            $composerLock = null;
+            $result = $this->tryFindVersionInComposerLock($composerLock);
```

Comment:
This seems like it will always send null as argument to tryFindVersionInComposerLock()?